accessors/clients/datastream/interface.go

```diff
@@ -60,3 +64,29 @@ func (c *DatastreamClientImpl) UpdateStream(ctx context.Context, req *datastream
 	}
 
 }
+
+func (c *DatastreamClientImpl) CreateConnectionProfile(ctx context.Context, createRequest *datastreampb.CreateConnectionProfileRequest) (*operation.OperationWrapper[datastreampb.ConnectionProfile], error) {
+	op, err := c.client.CreateConnectionProfile(ctx, createRequest)
```

Comment:
This seems to be slightly different from what we have been doing in other accessors, where the client interface has method signatures exactly equivalent to the accessors. Based on that logic, this would lie inside accessors.
However, these kind of functions would end up overloading the accessor and I think its better suited here how you have done it.

Reply:
Yes, I am dealing with the operation result here as well and not just making a call to the backend, so I think this should be more suitable